content/posts/2020-09-08-analiza-wynikow/index.md

```diff
@@ -0,0 +1,121 @@
+Pocieszającym może być fakt poprawy wyników w porównaniu z danymi sprzed 2 lat, jednak trzeba zauważyć, że poprawa jest niewielka. Wynik uśredniony podniósł się o zaledwie 0,2 punktu, zgodnie z wykresem poniżej:
+
+<div style="text-align:center;font-size:0.8em; margin-bottom:3em">
+  <img src="img\stare_wyniki.png"
+       alt="stare wyniki testu"
+       style="display:block;margin:0 auto;"
+  />
+  Wykres 2. Rozkład wyników testu z dnia 12.07.2018 (ilość odpowiedzi – 136).
+</div>
+
+## Najtrudniejsze z pytań
+
+Największy problem sprawiły pytania o numerach 8, 6 i 4. Ilość odpowiedzi poprawnych na pytania 8 i 6 była mniejsza niż ilość odpowiedzi błędnych. Dopiero w przypadku trzeciego miejsca tego niechlubnego podium sytuacja odwróciła się i poprawne odpowiedzi wyszły na prowadzenie, choć prawdę powiedziawszy z bardzo niewielką przewagą nad najczęściej udzielaną, błędną odpowiedzią.
+Najgorzej z całej trójki wypadło pytanie numer 8. Tutaj zaledwie nieco ponad 1/5 odpowiedzi została udzielona poprawnie. Widać więc, że największy problem w teście stanowi obsługa wyjątków w języku C++.
+
+<div style="text-align:center;font-size:0.8em; margin-bottom:3em">
+  <img src="img\pyt8a.png"
+       alt="Rozkład procentowy odpowiedzi na pytanie 8"
+       style="display:block;margin:0 auto;"
+  />
+  Wykres 3.1. Rozkład procentowy odpowiedzi na pytanie 8. Kolorem pomarańczowym oznaczono poprawną odpowiedź.</label>
+</div>
+
+Drugie miejsce w klasyfikacji najtrudniejszych pytań zajmuje pytanie szóste, które dotyczyło wskaźników. Z tym pytaniem poradzono sobie poprawnie zaledwie w 22% przypadków. Ponadto w tym przypadku poprawna odpowiedź jest dopiero na trzecim miejscu, za dwiema, najczęściej wybieranymi i błędnymi odpowiedziami.
+
+<div style="text-align:center;font-size:0.8em; margin-bottom: 3em">
+  <img src="img\pyt6a.png"
+       alt="Rozkład procentowy odpowiedzi na pytanie 6"
+       style="display:block;margin:0 auto;"
+  />
+  Wykres 3.2. Rozkład procentowy odpowiedzi na pytanie 6. Kolorem pomarańczowym oznaczono poprawną odpowiedź.
+</div>
+
+Brązowy medal w tej konkurencji zajmuje pytanie czwarte, którego treść dotyczyła typów zmiennych. Optymizmem napawa fakt, że w tym pytaniu najczęściej wybieraną odpowiedzią była odpowiedź poprawna, jednak niemal 2/3 odpowiedzi było odpowiedziami błędnymi.
+
+<div style="text-align:center;font-size:0.8em; margin-bottom: 3em">
+  <img src="img\pyt4a.png"
+       alt="Rozkład procentowy odpowiedzi na pytanie 4"
+       style="display:block;margin:0 auto;"
+  />
+  Wykres 3.3. Rozkład procentowy odpowiedzi na pytanie 4. Kolorem pomarańczowym oznaczono poprawną odpowiedź</label>
+</div>
+
+## Podsumowanie
+
+Przyglądając się wynikom i analizując je nasuwa się kilka wniosków.
+Można zauważyć, że osoby, które podjęły się wypełnienia testu mają zauważalne braki w wiedzy i umiejętnościach związanych z językiem C++ i programowaniem. Świadczą o tym liczne, błędne odpowiedzi.
```

Comment:
Ostatnio sam się szkolę z dobrego pisania tesktów i widzę u Ciebie tutaj trochę strony biernej. To bardzo przypomina sprawozdania. Co prawda charakter tego artykułu trochę taki jest, ale jeśli gdzieś uda Ci się pozbyć strony biernej to będzie miło :)